lib/ZnapZend.pm

```diff
@@ -600,8 +600,13 @@ my $sendRecvCleanup = sub {
             my $dstDataSet = $srcDataSet;
             $dstDataSet =~ s/^\Q$backupSet->{src}\E/$backupSet->{$dst}/;
 
-            $self->zLog->debug('sending snapshots from ' . $srcDataSet . ' to ' . $dstDataSet . ((grep (/^\Q$srcDataSet\E$/, @dataSetsExplicitlyDisabled)) ? ": not enabled, should be skipped" : ""));
-            {
+            my $srcDataSetDisabled = (grep (/^\Q$srcDataSet\E$/, @dataSetsExplicitlyDisabled));
+            $self->zLog->debug('sending snapshots from ' . $srcDataSet . ' to ' . $dstDataSet . ($srcDataSetDisabled ? ": not enabled, should be skipped" : ""));
```

Comment:
you are skipping, way "should" ?

Reply:
IIRC, copy-pasted from other skip messages, so kept in same style: note what we should do by hints from the world, eventually do take some step forward.

Reply:
Ok, wording addressed. 

Now I wonder if this message should only be seen `... if $self->debug` - configuration disablement is rather permanent and presumably intentional, so this has a chance to be regular and not particularly useful spam in daemon logs (journal, dmesg), unlike the earlier warnings about skips due to run-time hiccups (comms to backup server, inconsistent snapshot history, etc.).

Reply:
Or maybe not. We still post the snapshots we DO send, so this is just another line in the wall of text.